authentication/identity_providers/configuring-oidc-identity-provider.adoc

```diff
@@ -45,15 +45,25 @@ You can also indicate which claims to use as the user's preferred user name,
 display name, and email address. If multiple claims are specified, the first one
 with a non-empty value is used. The standard claims are:
 
-[horizontal]
-`sub`:: Short for "subject identifier." The remote identity for the user at the
+[cols="1,2"]
```

Comment:
Yeah I agree. I was just trying to make it better because I hated the way it was formatted kind of like a table, but with no borders. But headers would be good to add, so I'm updating at least the auth docs add headers and get rid of this weird syntax.